pycloudflare/models.py

```diff
@@ -191,36 +194,36 @@ def __repr__(self):
 
 class Record(object):
     _data = ()
+    _own_attrs = ('zone', '_service', '_data', '_old_data')
 
     def __init__(self, zone, data):
         self.zone = zone
         self._service = zone._service
         self._data = data
-        self._updates = {}
+        self._old_data = deepcopy(data)
```

Comment:
I'd be tempted to call that `_init_data` or `_saved_data`. "old" doesn't quite capture what's going on.